ipatests/test_webui/ui_driver.py

```diff
@@ -823,6 +823,41 @@ def del_multivalued(self, name, value, parent=None):
 
         assert clicked, 'Value was not removed: %s' % value
 
+    def undo_multivalued(self, name, value, parent=None):
+        """
+        Undo value in multivalued textbox.
+        """
+        if not parent:
+            parent = self.get_form()
+        s = "div[name='%s'].multivalued-widget" % name
+        w = self.find(s, By.CSS_SELECTOR, parent, strict=True)
```

Comment:
What is `w`?

Reply:
@stlaz I don't know what it is. It is used nearly everywhere in this file. Maybe widget? @pvoborni, could you please elaborate a bit? I chose it is widget.